src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Razor.Design": "1.0.0-*",
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-003890",
```

Comment:
Oh, yeah. Nit: use `1.0.0-preview3-*` so you don't have to keep manually updating.